tests/Stl.Fusion.Tests/Server/ExceptionContextFixture.cs

```diff
@@ -0,0 +1,87 @@
+﻿#if NETFRAMEWORK
+using System.Web.Http.Filters;
+using System.Web.Http.Controllers;
+using System.Web.Http;
+using System.Web.Http.Dependencies;
+#else
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+#endif
+
+using Moq;
+using AutoFixture;
+using Stl.Fusion.Server;
+
+namespace Stl.Fusion.Tests.Server;
+
+public class ExceptionContextFixture : BaseFixture<
+#if NETFRAMEWORK
+    HttpActionExecutedContext
+#else
+    ExceptionContext
+#endif
+>
+{
+    private Mock<ILogger<JsonifyErrorsAttribute>> _loggerMock = new();
+    private Mock<IErrorRewriter> _errorRewritterMock = new();
+#if NETFRAMEWORK
+    private Mock<IDependencyResolver> _serviceProviderMock = new();
+#else
+    private Mock<IServiceProvider> _serviceProviderMock = new();
+    private Mock<IActionResultExecutor<ContentResult>> _actionResultExecutorMock = new();
+#endif
+
+    public ExceptionContextFixture()
+    {
+        _errorRewritterMock.Setup(x =>
+                x.Rewrite(It.IsAny<object>(), It.IsAny<Exception>(), It.IsAny<bool>()))
+            .Returns((object _, Exception x, bool _) => x);
+
+        _serviceProviderMock
+            .Setup(x => x.GetService(typeof(IErrorRewriter)))
+            .Returns(_errorRewritterMock.Object);
+
+        _serviceProviderMock
+            .Setup(x => x.GetService(typeof(ILogger<JsonifyErrorsAttribute>)))
+            .Returns(_loggerMock.Object);
+
+        Fixture.Register(() => _serviceProviderMock);
```

Comment:
Everything required to create a valid object is in the constructor. Applying modifications to the desired output object use the builder pattern.